server.ts

```diff
@@ -51,7 +51,9 @@ const CONFIG = {
   'appInsights_enabled': process.env['APPINSIGHTS_ENABLED'] || 'false',
   'appInsights_roleName': process.env['APPINSIGHTS_ROLE'] || 'ccd-case-management-web',
   'shutter_redirect_url': process.env['SHUTTER_REDIRECT_URL'] || '',
-  'shutter_redirect_wait': parseInt(process.env['SHUTTER_REDIRECT_WAIT'], 10) || 10
+  'shutter_redirect_wait': parseInt(process.env['SHUTTER_REDIRECT_WAIT'], 10) || 10,
+  'logging_level': process.env['LOGGING_LEVEL'] || 'Off',
```

Comment:
`Off` doesn't read like a logging level. Perhaps change var name?

Reply:
Removed 'Off' from the logging level.